.github/ISSUE_TEMPLATE/-hammer--feature-request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: ":hammer: Feature request"
+about: Feature request
+title: ":hammer: Feature: "
+labels: enhancement
+assignees: ''
+
+---
+
+<!-- Remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm the following points: -->
+<!-- (it should look like this: - [x] I have ...) -->
+- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
+- [ ] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated
+- [ ] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
+
+## Describe the feature that is missing
```

Comment:
perhaps just `## Describe the feature`? I think the current wording may discourage people from making feature requests for new functionality that's not necessarily directly related to FFXI gameplay